OlesiaBarannik/HW6/2.py

```diff
@@ -0,0 +1,14 @@
+# Task 2. Write a script that validates the login that the user enters.
+# If the login is "First", then welcome users. If there is a login
+# else, send an error message.
+# (need to use a while loop)
+
```

Comment:
good job
look at this implementation

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)